.github/workflows/publish-study-definition.yml

```diff
@@ -34,7 +34,7 @@ on:
         default: staging
 
 concurrency:
-  group: production
+  group: ${{ inputs.environment || 'staging' }}
```

Comment:
Have to double check the syntax here if this works if there is no input but if that's correct 👍